File: 2019-x-mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int piramida(string prompt);

int main(void)  
{
    int i = piramida("Height: ");
    printf("Store: %i\n", i);
    for (int x = 0; x < i; x++)
    {
        for (int y = x; y < i; y++)
        {
            printf(" ");
        }
        for (int y = x; y > 0; y--)
        {
            printf("#");
        }
         printf("#\n");
    }
}


int piramida(string prompt)
{
    int n;
    do
    {
         n = get_int("%s", prompt);
    }
    while (n < 1 || n>8);
    return n;
}

File: 2020-x-sentimental-mario-more/mario.py
from cs50 import get_int
from sys import exit

while True:
    x = get_int("Height: ")

    if x < 1 or x > 8:
        print("Not correct input")
    else:
        for i in range(x):
            print(" " * (x-i-1), end = "")
            print("#" * (i + 1), end =" ")
            print("#" * (i + 1))
        exit(0)
